### PR TITLE
[faraland-staking] update subgraph URL

### PR DESCRIPTION
**Pull request by tangobeo**
Fixes # .

Changes proposed in this pull request:
- update subgraph URL

### CODE REVIEW

**Comment by ChaituVR** [comment]
@edwardEvans094 Could you confirm this change ?

**Comment by ChaituVR** [comment]
@toniqhz  Could you confirm this change ?

File: src/strategies/faraland-staking/index.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ import { getAddress } from '@ethersproject/address';
 import { subgraphRequest } from '../../utils';
 
 const FLASHSTAKE_SUBGRAPH_URL = {
-  '1': 'https://queries-graphnode.faraland.io/subgraphs/name/edwardevans094/farastore-v12',
-  '56': 'https://queries-graphnode.faraland.io/subgraphs/name/edwardevans094/farastore-v12'
+  '1': 'https://queries-graphnode-x.faraland.io/subgraphs/name/edwardevans094/farastore-v1',
+  '56': 'https://queries-graphnode-x.faraland.io/subgraphs/name/edwardevans094/farastore-v1'
 };
 
 export const author = 'edwardEvans094';
```
Comment (ChaituVR) [inline src/strategies/faraland-staking/index.ts:9]:
```suggestion
export const author = 'toniqhz';
```

Comment (toniqhz) [reply src/strategies/faraland-staking/index.ts:9]: its ok, pls merge

**Review by ChaituVR — CHANGES_REQUESTED** [review]
@tangobeo test cases are failing, can take a look? 

image

You need to pass a address that has voting power with latest block

**Comment by tangobeo** [comment]
> @tangobeo test cases are failing, can take a look?
> 
> image
> You need to pass a address that has voting power with latest block

I just update it, please check again. Thank you!

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**